src/Commands/pm/ThemeCommands.php

```diff
@@ -55,4 +94,33 @@ public function uninstall(array $themes): void
         $this->getThemeInstaller()->uninstall($themes);
         $this->logger()->success(dt('Successfully uninstalled theme: !list', ['!list' => implode(', ', $themes)]));
     }
+
+    public function addInstallDependencies($themes): array
```

Comment:
Needs Doxygen

Reply:
Oh, I didn't know there was a 14.x, I went straight with the default one. Thanks.